src/validation.cpp

```diff
@@ -5520,7 +5520,7 @@ Chainstate& ChainstateManager::InitializeChainstate(CTxMemPool* mempool)
     const bool destroyed = DestroyDB(path_str);
 
     if (!destroyed) {
-        LogPrintf("error: leveldb DestroyDB call failed on %s\n", path_str);
+        LogError("[snapshot] leveldb DestroyDB call failed on %s\n", path_str);
```

Comment:
I don't think this should be `LogError` as this can be recovered from, e.g. [here](https://github.com/bitcoin/bitcoin/blob/bd642ee15bda313bcf801cf63e4428c7a7d252c8/src/validation.cpp#L6332). Ideally, we just return error messages here and let the callsite log - but barring that I'd just leave as is since all the call sites that do lead to node termination will already log on the error level?